cfme/tests/containers/test_containers_properties.py

```diff
@@ -40,7 +43,8 @@ def test_pods_properties_rel(provider, rel):
     the fields in the Properties table
     """
     navigate_to(Pod, 'All')
-    ui_pods = [r.name.text for r in list_tbl_pods.rows()]
+    ui_pods = sample([r.name.text for r in list_tbl_pods.rows()],
```

Comment:
I agree, BTW, this behavior of choose N random samples repeats many times (at least in our tests and I'm sure that in many other) so maybe it worth to do something public? maybe in common or something and refactor accordingly?